Makefile

```diff
@@ -26,13 +26,20 @@ NANOS_ID = 1
 WORDS = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about"
 PIN = 5555
 
+# pool operator app contains additional functionality
+# set to 1 if you want to include it, or to 0 if you want a regular app
+POOL_OPERATOR_APP = 1
```

Comment:
we can already get rid of this as we ended up agreeing on not splitting the app (yet)

Reply:
ok I see it's getting removed in the other PR